health-check/src/uyuni_health_check/config.py

```diff
@@ -0,0 +1,78 @@
+import os
+from typing import Any, Dict
+import tomli
+from pathlib import Path
+import json
+from jinja2 import Environment, FileSystemLoader, Template
+
+BASE_DIR = os.path.dirname(os.path.abspath(__file__))
+CONFIG_DIR = os.path.join(BASE_DIR, "config")
+TEMPLATES_DIR = os.path.join(CONFIG_DIR, "templates")
+CONTAINERS_DIR = os.path.join(BASE_DIR, "containers")
+CONFIG_TOML_PATH = os.environ.get("HEALTH_CHECK_TOML", os.path.join(BASE_DIR, "config.toml"))
+
+def get_json_template_filepath(json_relative_path: str) -> str:
+    return os.path.join(TEMPLATES_DIR, json_relative_path)
+
+def load_jinja_template(template: str) -> Template:
+    return _conf.jinja_env.get_template(template)
+
+def load_dockerfile_dir(dockerfile_dir: str) -> str:
+    return os.path.join(CONTAINERS_DIR, dockerfile_dir)
+
+def get_config_dir_path(component: str) -> str:
+    return os.path.join(CONFIG_DIR, component)
+
+def load() -> Dict:
+    return _conf.parsed_config
+
+def load_prop(property: str) -> Any:
+    res = _conf.parsed_config.copy()
+    for prop_part in property.split('.'):
+        try:
+            res = res.get(prop_part, {})
+        except (AttributeError, ValueError):
+            res = None
```

Comment:
Does it make sense to turn access errors into `None`? This might silence errors caused by bad callers (or bad configs)

Reply:
The caller always needs to deal with the case that it asks for something from the config that can't be found  (`load_prop(x)` returns `None`). 

The question is: do we force the caller to deal with it by raising an exception or are we okay with the caller ignoring that? Both ways are okay, but since we don't have any config validation, it might make sense to not silence lookup errors (e.g. because the config is incomplete and mandatory values are missing) at runtime

Reply:
Ack, makes sense. Fixed - instead of silencing the error, we are now  explicit as to the input we wanted to get, where we failed, and the state of config.